test/libsolidity/util/SoltestTypes.h

```diff
@@ -125,6 +127,11 @@ struct ABIType
 	Type type = ABIType::None;
 	Align align = ABIType::AlignRight;
 	size_t size = 32;
+
+	// note: bits & fractionalDigits are right now only used FixedPointType.
+	size_t bits = 0;
+	size_t fractionalDigits = 0;
```

Comment:
```suggestion
	/// only used for UnsignedFixedPoint and SignedFixedPoint
	size_t bits = 0;
	/// only used for UnsignedFixedPoint and SignedFixedPoint
	size_t fractionalDigits = 0;
```

Reply:
Oh but we said that we remove `bits` right?

Reply:
Ok, I remove `bits` now. If we need that information later, we can just add it later.